backend/src/main/java/gov/cdc/usds/simplereport/db/model/PersonUtils.java

```diff
@@ -709,7 +709,7 @@ public static Map<String, String> tribalMap() {
           "not_hispanic", not_hispanic,
           "not hispanic or latino", not_hispanic,
           "2186-5", not_hispanic,
-          "refused", List.of("U", "unknown"));
+          "refused", List.of(MappingConstants.UNK_CODE, MappingConstants.UNKNOWN_STRING));
```

Comment:
I'm not sure about this change because the code system we're using when we build the observation [here](https://github.com/CDCgov/prime-simplereport/blob/6e2d66b02299d619221dd87c79c5738bb06a4f42/backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java#L378-L378) is https://terminology.hl7.org/6.2.0/CodeSystem-v2-0189.html which uses "U"